### PR TITLE
chore: Add hero fields to seed.js

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-224

### Related PRs
* https://github.com/USSF-ORBIT/ussf-portal-client/pull/862
* https://github.com/USSF-ORBIT/ussf-portal-cms/pull/98

## Proposed changes

* Add hero db fields to seed.js
* Fix Label.Type errors (unclear why they just started...)
* Add placeholder env vars to docker compose file
* Try (and fail) to write a test for uploading hero image

## Setup

* In `ussf-portal/e2e` dir, run `yarn services:up`
* Run `yarn e2e:test` for Playwright tests
* Run `yarn cypress:dev` for Cypress tests
* Make sure all tests pass

## Review Notes

I haven't figured out how to write a test that uploads a hero image. Playwright provides functionality for this, but it relies on one of two things:

1. **An input element with type=file.** Unfortunately, Keystone renders the Upload button as a _button_, not an input element. Doesn't work.
2. **A filechooser event to be fired after clicking Upload** Unfortunately,  a `filechooser` event is never fired after clicking Upload Image. 😩 

Docs here: https://playwright.dev/docs/input#upload-files

### CODE REVIEW

File: e2e/playwright/database/seed.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ const dropAndCreateSchema = async (client) => {
     "name" text NOT NULL DEFAULT ''::text,
     "isAdmin" bool NOT NULL DEFAULT false,
     "isEnabled" bool NOT NULL DEFAULT false,
+
```
Comment (gidjin) [inline e2e/playwright/database/seed.js:39]: Question: did you plan to add something here?

File: e2e/playwright/database/seed.js [hunk 2]
```diff
@@ -99,9 +105,11 @@ ALTER TABLE "public"."User" ADD FOREIGN KEY ("updatedBy") REFERENCES "public"."U
     "createdBy" text,
     "updatedAt" timestamp(3),
     "createdAt" timestamp(3) DEFAULT CURRENT_TIMESTAMP,
+
```
Comment (gidjin) [inline e2e/playwright/database/seed.js:108]: Question: Is this to help with readability?

**Review by gidjin — APPROVED** [review]
Changes look good. Reminder to timebox looking at a test for hero image, I'm not too worried if that's the only thing we can't get to work. Maybe see if we can trigger the event if keystone isn't? I'm approving since I trust you to know when to stop the timebox.

File: e2e/playwright/database/seed.js (hunk 2 above)
Comment (abbyoung) [reply e2e/playwright/database/seed.js:108]: i think these were mistakes/artifacts of troubleshooting, will revert 👍

**Pull request merged**